src/components/homepage/homepage-oer-presentation.yml

```diff
@@ -0,0 +1,17 @@
+homepageOerPresentation:
+  headline:
+    en: Become part of the OER community!
+    de: Werde Teil der OER-Community!
+  descriptionMarkdown:
+    en: |
+      Our goal is to create a community project that is carried and supported by as many people around the world as possible.
+      If you are inspired by the idea of free learning and of an open learning platform, then join our community for **free**.
+    de: |
+      Unser Ziel ist, ein Gemeinschaftsprojekt zu schaffen, das von möglichst vielen Menschen auf der ganzen Welt getragen und unterstützt wird.
+      Wenn du von der Idee des kostenlosen lernens und einer offenen Lernplattform begeistert bist uns selbst Inhalte generieren möchtest, trit unserer Community **kostenlos** bei.
```

Comment:
Wenn du von der Idee des kostenlosen **L**ernens und einer offenen Lernplattform begeistert bist un**d** selbst Inhalte generieren möchtest, trit**t** unserer Community kostenlos bei

_(corrections in bold)_

Reply:
Thank you! I have corrected everything you found 🙏

Reply:
:+1: